### PR TITLE
add submariner-addon

### PR DESCRIPTION
**Pull request by ldpliu**
https://github.com/open-cluster-management/backlog/issues/6433

### CODE REVIEW

File: ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-master.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: submariner-addon
+promotion:
+  name: master
```
Comment (qiujian16) [inline ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-master.yaml:17]: i do not think we promote to master

File: ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-master.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  namespace: open-cluster-management
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.4"
```
Comment (petr-muller) [inline ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-master.yaml:27]: Are you sure about using `4.4` for your testing?

Comment (ldpliu) [reply ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-master.yaml]: Thanks, I have updated to 4.6

**Review by petr-muller — APPROVED** [review]

**Comment by qiujian16** [comment]
/hold
I think  you  need to squash

**Comment by ldpliu** [comment]
> /hold
> I think you need to squash

done

**Pull request merged**